spec/schedulers/VirtualTimeScheduler-spec.ts

```diff
@@ -61,43 +61,9 @@ describe('VirtualTimeScheduler', () => {
     expect(invoked).to.deep.equal([6, 4, 1, 3, 5, 2]);
   });
 
-  it('should support recursive scheduling', () => {
-    const v = new VirtualTimeScheduler();
-    let count = 0;
-    const expected = [100, 200, 300];
-
-    v.schedule<string>(function (this: SchedulerAction<string>, state: string) {
-      if (++count === 3) {
-        return;
-      }
-      const virtualAction = this as VirtualAction<string>;
-      expect(virtualAction.delay).to.equal(expected.shift());
-      this.schedule(state, virtualAction.delay);
-    }, 100, 'test');
-
-    v.flush();
-    expect(count).to.equal(3);
-  });
```

Comment:
Was this removed before the `rescheduled` parameter was added? Should rescheduling be supported here?